sdp-utils.h

```diff
@@ -78,7 +78,7 @@ const char *janus_sdp_mtype_str(janus_sdp_mtype type);
 /*! \brief Helper enumeration to quickly identify m-line directions */
 typedef enum janus_sdp_mdirection {
 	/*! \brief default=sendrecv */
-	JANUS_SDP_DEFAULT,
+	JANUS_SDP_DEFAULT = 0,
```

Comment:
This also is unrelated

Reply:
It's just to be 100% sure that JANUS_SDP_DEFAULT will be equal zero.

Reply:
C already guarantees this.

Reply:
It is, but someone can change it to e.g. -2. There are such enums in Janus. I make it start from 0 just to show it must start from 0 and remove code something like some field = JANUS_SDP_DEFAULT because of g_malloc0.